apps/api/src/domain/models/audio-item/commands/transcripts/add-participant-to-transcript/add-participant-to-transcript.command-handler.ts

```diff
@@ -0,0 +1,70 @@
+import { CommandHandler } from '@coscrad/commands';
```

Comment:
Be sure to register the command and its handler in the appropriate domain module.

Reply:
done